src/pages/jonghoLee/MainPage/Main/MainLeft/Article.js

```diff
@@ -0,0 +1,98 @@
+import React, { Component } from 'react';
+import UserComment from './UserComment';
+import Comment from './Comment';
+import './Article.scss';
+import moreImg from '../../../../../images/jonghoLee/more.png';
+import heartImg from '../../../../../images/jonghoLee/heart.png';
+import chatImg from '../../../../../images/jonghoLee/chat.png';
+import paperImg from '../../../../../images/jonghoLee/paper-plane.png';
+import markImg from '../../../../../images/jonghoLee/small-bookmark.png';
+
+export default class Article extends Component {
+  constructor() {
+    super();
+
+    this.state = {
+      comments: [],
+    }
+  }
+
+  createComment = (comment) => {
+    let commentObj = {
+      comment,
+      isLiked: false
+    };
+
+    this.setState({
+      comments: [...this.state.comments, commentObj],
+    })
+  }
+
+  changeHeart = (index) => {
+    let { comments } = this.state;
+
+    this.setState({
+      comments: comments.map((commentObj, i) => {
+        if (i === index) {
+          commentObj.isLiked = true;
+        }
+        return commentObj
+      })
+    })
+  }
```

Comment:
```suggestion
  changeHeart = (selectedNum) => {
    const { comments } = this.state;
    const newComments = comments.map(comment => {
        if (comment.id === selectedNum) {
          comment.isLiked = true;
        }
        
        return commentObj
      })

    this.setState({
      comments: newComments
    })
  }
```
- this.state.comments 구조 분해 할당하는 경우 let 키워드를 사용하신 이유가 있을까요?
- index를 활용하기 보다는 comment 하나의 고유 id를 사용해주는 것이 좋습니다. (index는 순서에 따라서 달라질 수 있겠죠?)
- 인자가 길어지는 경우 변수를 활용해서 깔끔하게 처리해보세요! 가독성이 좋아질 수 있습니다.